qiskit/opflow/gradients/circuit_gradients/lin_comb.py

```diff
@@ -119,13 +120,22 @@ def convert(
                     If a Tuple[ParameterExpression, ParameterExpression] or
                     List[Tuple[ParameterExpression, ParameterExpression]]
                     is given, then the 2nd order derivative of the operator is calculated.
-
+            aux_meas_op: The operator that the auxiliary qubit is measured with respect to.
+                         for aux_meas_op = Z we compute Re[(dω⟨ψ(ω)|)O(θ)|ψ(ω)〉]
+                         for aux_meas_op = -Y we compute Im[(dω⟨ψ(ω)|)O(θ)|ψ(ω)〉]
         Returns:
             An operator corresponding to the gradient resp. Hessian. The order is in accordance with
             the order of the given parameters.
+        Raises:
+            Warning: If the provided auxiliary measurement operator acts on more that 1 qubit.
         """
-
-        return self._prepare_operator(operator, params)
+        if aux_meas_op.num_qubits != 1:
+            raise Warning(
```

Comment:
I think this should be an error since it's the wrong input value instead of a warning 
```suggestion
            raise ValueError(
```